recipes/expam/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "expam" %}
+{% set version = "1.0.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 2f44ead7b7bd34821cfe495e8fc6296e14786c55d5730d83ee90d1e38ab56b12
+
+build:
+  number: 0
+  skip: true  # [py < 38]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - pip
+    - python
+    - Cython ==0.29.28
+    - numpy ==1.22.2
+    - setuptools
+  run:
+    - python
+    - numpy ==1.22.2
+    - matplotlib-base ==3.5.1
+    - pandas ==1.4.1
+    - psutil ==5.9.0
+    - requests ==2.27.1
+    - pytables ==3.7.0
```

Comment:
Does it actually require these exact versions? That will make it almost immediately incompatible with other packages.

Reply:
Thanks for that @dpryan79, I modified the requirements and reduced it down to something less restricting. The new version seems to pass all the tests (c6c15d1 below).